json/sle-15-64bit.json

```diff
@@ -2,5 +2,11 @@
     "uuid": "b5915805-d800-459d-826a-322dc9cf07e2",
     "reference_label": "sle-15-64bit",
     "name_label": "SUSE Linux Enterprise 15 (64-bit)",
-    "derived_from": "base-sle-hvm-64bit.json"
+    "derived_from": "base-linux-uefi.json",
```

Comment:
This loses the `name_description` that it previously inherited from `base-sle-hvm.json`.

It might be worth revisiting the name descriptions at some point since I'm not sure if they are relevant or useful. Nevertheless, it shouldn't be changed as part of an unrelated change.

Reply:
This template will inherit a new `name_descirption ` from `base-hvmlinux.json`, which include a more general description of the templates. and this parameter is just a description of this templates, it will not involve XC, and it is not visible to users as well, So.. think it`s ok?

Reply:
Certainly, I could provide a more specific `name_description` for this template, but doing so would require creating a new one like `base-linux-uefi-sle.json`, which complicated the inheritance process. so, I prefer to keep it for now.

Reply:
Why is it not possible to copy the `name_description` from `base-sle-hvm.json` directly into this template file like you copied other parameters such as `max_memory`?

Reply:
Yeah, I can copy the `name_description` from `base-sle-hvm.json` to this template, but the `name_description ` in the base-sle-hvm.json is a general description. Putting it into a detailed template seems a bit weird. 
Maybe a better approach is to add a `name_description `in the `base-uefi-linux.json`?

Reply:
I guess my point was to avoid changing unrelated parts of the templates as a side effect of this change. Putting the name description into `base-uefi-linux.json` would then cause other templates' name descriptions to change.

Anyway, I'll approve this - I don't think anyone ever looks at the name description so it probably doesn't matter too much.

Reply:
I think Xen Orchestra uses the name description to pre-fill the description field at VM creation.

Reply:
Confirmed.

Reply:
it will inherit a more general `name_description`, I think it`s fine.